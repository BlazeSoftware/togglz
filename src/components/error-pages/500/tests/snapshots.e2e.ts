import snapIt from '../../../../test/snap-it';

describe('500', () => {
  snapIt('app-broken')('renders correctly', '<app-broken></app-broken>');
});
