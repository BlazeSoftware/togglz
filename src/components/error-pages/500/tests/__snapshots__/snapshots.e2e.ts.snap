// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`500 renders correctly 1`] = `"<app-broken class=\\"hydrated\\"><div class=\\"o-container o-container--small u-window-box-xlarge u-centered\\"><stencil-route-title class=\\"hydrated\\"></stencil-route-title><h2 class=\\"c-heading\\">Sorry, something went wrong.</h2><div><a class=\\"c-link\\" href=\\"/\\">Go back to the home page</a></div></div></app-broken>"`;
